Updating FPG standard deviation to be NHANES based (#1145)
ihmeuw/vivarium_research: docs/source/models/risk_exposures/fpg/index.rst

values, we imposed a maximum of 30 and minimum of 1 mmol/L. As we need to update to GBD 2021 we are 
using this as a solution until updating. 

Another noticed issue with GBD data is striations in the standard deviation values. The graph below 
visually shows this issue. After speaking with GBD modelers, it seems that these are known issues 
and will not be addressed in GBD 2021. These striations are causing higher than expected FPG exposures. 
To account for this, we are instead using standard deviations derived from NHANES data. 

.. image:: FPG_SD.png

> To create these new standard deviation draws, NHANES data was bootstrapped to a subset of the original 

Reviewer: ```suggestion
To create these new standard deviation draws, we bootstrapped the NHANES data to a random resampling of the original 
```

> dataset, and then standard deviations were calculated by age/sex. The full workbook can be `found here <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/fpg_std_nhanes.R>`_

Reviewer: ```suggestion
dataset, and then calculated the standard deviations by age/sex. The full workbook can be `found here <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/fpg_std_nhanes.R>`_
```
